.eslintrc.json

```diff
@@ -1,14 +1,26 @@
 {
     "extends": ["eslint:recommended", "google"],
-    "plugins": ["prettier"],
+    "plugins": ["prefer-arrow"],
```

Comment:
By now it may be incredibly hard to untangle, but doing the reformatting resulting from this and other linter changes separately from the backend overhaul would be great.

What I'd do: copy over the relevant changes to package.json and .eslintrc.json changes to a new branch, and reformat using `--fix` and cherry-picking over any changes that can't be made automatically.

When that changes is on main, merge in the main branch and resolve any conflicts in favor of what's on this branch. What remains should be the real changes on this branch.

Reply:
Cherry-picked as #421!

Reply:
Thanks!

Reply:
Done in https://github.com/foolip/mdn-bcd-collector/pull/421, thanks!